tests/e2e/server.php

```diff
@@ -30,7 +31,8 @@
 App::get('/redirect')
     ->inject('response')
     ->action(function($response) {
-        /** @var Utopia/Response $response */
+        /** @var Utopia /Response $response
+         * @phpstan-ignore-next-line */
```

Comment:
This should probably be:

```suggestion
    ->action(function (Response $response) {
```

Reply:
Thanks, will apply the suggested change along with all others after you confirm for all the issues above.